src/assets/scripts/client/navigationLibrary/FixCollection.js

```diff
@@ -101,6 +100,28 @@ class FixCollection extends BaseCollection {
         return fixModel.relativePosition;
     }
 
+    /**
+     * Returns the nearest fix to the specified position
+     *
+     * @for FixCollection
+     * @method getNearestFix
+     * @param position {StaticPositionModel}
+     * @param allowRnavFixes {boolean} A flag indicating whether RNAV fixes should be used
+     */
+    getNearestFix(position, allowRnavFixes = false) {
```

Comment:
`allowRnavFixes` --> `hiddenFixes`

(RNAV fixes are totally game, just not the fake ones we create to make certain procedures work, so I don't want to confuse readers of the code when they see that we aren't including "RNAV fixes", when really we _are_ 😉)

Reply:
Hi hadn't considered that. :+1: